lib/Backends/NNPI/tests/NNPIOperatorTest.cpp

```diff
@@ -323,10 +323,6 @@ struct BlacklistInitializer {
             {"RoiAlignRotatedBatchIndexInBoxesTensor/0",
              TestBlacklist::AnyDeviceAnyEngine},
             {"FP16RoiAlign/0", TestBlacklist::AnyDeviceAnyEngine},
-            {"FP16RoiAlignWithAlignedCoordinates/0",
```

Comment:
please rebase over master and instead of completely deleting these move them into this ifdef (it already exists):
#if NNPI_MAJOR_VERSION == 1 && NNPI_MINOR_VERSION == 0